docs/resources/deployment-examples/example-fargate/readme.md

```diff
@@ -0,0 +1,85 @@
+```
+[TODO replace deployment aws policy]
+```
+#Hosted Zone and ACM Certificates
+This example requires a hosted zone to enable public access to the SEP service.  You will need to create the hosted zone separately and then configure the `hosted_zone_name` terraform variable before running plan/apply.   You will also need to ensure your sample configuration that uses the hosted zone domain name is updated appropriately. 
+
+# Terrafrom Workspace Variables
+The following workspace variables need to be set befpre you plan/apply.
+
+| variable name | type | description | 
+|---|---|---|
+| environment | plain text | deployment environment. resources will be prefixed | 
+| hosted_zone_name | 
+| sqs_secret_key | SENSITIVE | AWS Secret Key of SQS Iam User |	
+| sqs_access_key | SENSITIVE | AWS Access Key of SQS Iam User | 
+| sqlite_password | SENSITIVE	| Database password |
+| sqlite_username | SENSITIVE | Database username |
+| sep10_signing_seed | SENSITIVE | Sep10 Transaction verification |
+| jwt_secret | SENSITIVE | Sep10 Jwt encryption key | 
+| AWS_SECRET_ACCESS_KEY | SENSITIVE | Terraform Deployment AWS User Secret Key |
+| AWS_ACCESS_KEY_ID | SENSITIVE | Terraform Deployment AWS User Access Key Id |
+
+# Secrets Management
+Anchor Platform Config allows you to specify placeholders for variables that contain secrets. The placeholder names will be read from environment variables at runtime.  In this example, secrets are provisioned as container definition environment variables from AWS Parameter Store.  The secrets are provisioned to the parameter store using terraform and read from encrypted terraform workspace variables. 
+
+# ECS Clusters
+There are two ECS Clusters in this example.  The `SEP Service` runs in one cluster to support the front end `SEP-31 protocol` service.  The `Reference Service` cluster hosts an example receiving anchor backend service.   Both services include a load-balancer which calles `/health` endpoints to determine listener target health. Both clusters are deployed in a `private subnet`. The `SEP Service` loadbalancer is deployed in a public subnet to enable public internet access. As a back-end service, the `Reference Service` loadbalancer is deployed in `private` subnet.
+
+<img width="971" alt="image" src="https://user-images.githubusercontent.com/98427531/179147580-b9da19ab-ea05-4a08-ad9a-868be8d3442b.png">
+
+There are two ECS Clusters in this example.  The `SEP Service` runs in one cluster to support the front end `SEP-31 protocol` service.  The `Reference Service` cluster hosts an example receiving anchor backend service.   Both services include a load-balancer which calles `/health` endpoints to determine listener target health. Both clusters are deployed in a `private subnet`. The `SEP Service` loadbalancer is deployed in a public subnet to enable public internet access. As a back-end service, the `Reference Service` loadbalancer is deployed in `private` subnet.
```

Comment:
This paragraph looks like a duplicate (same text appears 2 paragraphs above)

Reply:
ill be darned. thanks!